kube-runtime/src/controller/mod.rs

```diff
@@ -69,6 +69,8 @@ impl Action {
     /// even in the case of missed changes (which can happen).
     ///
     /// Watch events are not normally missed, so running this once per hour (`Default`) as a fallback is reasonable.
+    ///
+    /// The duration is clamped to a maximum of 6 months.
```

Comment:
The function was always best-effort, I don't think we should guarantee a specific time in the API.

```suggestion
```